db/schema.rb

```diff
@@ -11,10 +11,6 @@
 # It's strongly recommended that you check this file into your version control system.
 
 ActiveRecord::Schema.define(version: 2021_08_31_213231) do
-<<<<<<< HEAD
```

Comment:
Someone already removed this in their own PR, so watch out for merge conflicts after merging either this one or the other!